src/saturn_engine/worker/topics/rabbitmq.py

```diff
@@ -69,6 +69,7 @@ class Options:
         prefetch_count: t.Optional[int] = None
         serializer: RabbitMQSerializer = RabbitMQSerializer.JSON
         log_above_size: t.Optional[int] = None
+        shared_lock_max_reservations: int = 8
```

Comment:
This is a public option, make it actually meaningful -> `max_publish_concurrency`

Reply:
Done